### ISSUE TITLE
Factors that improved the performance on RTE and SQuAD 2.0.

### ISSUE THREAD

**Issue by GeondoPark** [huawei-noah/Pretrained-Language-Model#95]
Hi, huawei-noah team.

Thank you for sharing the code of your interesting work, TinyBERT. 
I wonder which factors resulted in the performance improvement on the RTE and SQuAD 2.0 datasets, comparing the previous and recent versions. 

Thanks.

**Comment by XiaoqiJiao** [comment]
Hi, Geondo.

The performance on RTE is unstable due to the small data set, so we just finetuned a few more times.

For SQuAD 2.0, we found that performing the intermediate layer distillation on the augmented task data set for more epochs is helpful.

**Comment by GeondoPark** [comment]
Thank you for your response.

I wondering which part you finetuned for improving the performance of RTE. (learning rate, epochs, temperature, intermediate layer coefficient, and so on..! )
